assets/css/admin/chosen/_colors.scss

```diff
@@ -0,0 +1,33 @@
+
+/* WordPress Color Schemes */
+
+:root {
+	@include admin-scheme(#007cba);
+}
+@include wordpress-admin-schemes();
+
+body.admin-color-evergreen {
+	@include admin-scheme(#36533f);
+}
+
+body.admin-color-mint {
+	@include admin-scheme(#4f6d59);
+}
```

Comment:
Perhaps we should make our own mixin that ensures the fallback default colors are set for CSS variable support, includes the core color schemes, as well as our additional bbPress color scheme support.

Reply:
Just created `@mixin edd-admin-colors` along with adding bbPress colors to the color variables. New color theme implementation is now just:

```css
:root {
	@include edd-admin-colors();
}
```
Which maybe should be in the main EDD admin stylesheet, rather than each individual one, but it is in the Chosen one right now.